pykern/pkcli/test.py

```diff
@@ -104,7 +104,12 @@ def _find(paths):
     i = re.compile(r'(?:_work|_data)/')
     res = []
     cwd = pkio.py_path()
-    for t in paths or ('tests',):
+    d = paths
```

Comment:
This temporary is not necessary. `paths` is a variable taht can be falsey, and if it is, then you are setting it to a default value. The extra variable `d` just adds unnecessary complexity. Similarly for `cwd` which is unnecessary.

Reply:
```python
d = paths
``` 
is outdated. Why is cwd unnecessary?

Reply:
I see now, never mind.